src/main/java/com/linecorp/armeria/server/http/file/MimeTypeUtil.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.file;
+
+import static java.util.Objects.requireNonNull;
+
+import java.net.URLConnection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+final class MimeTypeUtil {
+
+    /**
+     * A map from extension to MIME types, which is queried before
+     * {@link URLConnection#guessContentTypeFromName(String)}, so that
+     * important extensions are always mapped to the right MIME types.
+     */
+    private static final Map<String, String> EXTENSION_TO_MIME_TYPE;
+
+    static {
+        final Map<String, String> map = new HashMap<>();
+
+        // Text files
+        add(map, "text/css", "css", "scss", "less");
```

Comment:
`scss` and `less` files are not meant to be exposed in the browser directly, as I know. They are compiled into `css` files.